src/type-extensions.ts

```diff
@@ -121,6 +121,21 @@ declare module "hardhat/types/runtime" {
             address?: string,
             networkId?: string
         ) => Promise<LoadL1MessagingContractResponse>;
+
+
+        /**
+         * Increases the block timestamp offset.
+         * @param seconds the new timestamp in seconds
```

Comment:
This JSDoc is wrong. It is not the timestamp which is is set, but the offset which is increased.